party_time/party_time.lua

```diff
@@ -109,7 +128,7 @@ command.arg.register_type('lookup_option', {
 })
 
 -- Addon Command Handlers
--- Seven main commands additoinal sub_commands within
+-- Eleven main commands additoinal sub_commands within
```

Comment:
Not that I mind, but why mention the amount it in the comment? :D Only forces you to keep it up to date.